schemas/deviceShadow/ipShadow/ipShadow.json

```diff
@@ -171,9 +171,9 @@
                     "type": "integer"
                 },
                 "mccmnc": {
-                    "type": "string",
+                    "type": "number",
```

Comment:
the modem_info library provides a string for this value.
the nrf_cloud library reports this value as a string.

Reply:
Current asset_tracker_v2 sends number: 

```json
"roam": {
          "v": {
            "band": 12,
            "nw": "LTE-M",
            "rsrp": -119,
            "area": 34635,
            "mccmnc": 310410,
            "cell": 141255696,
            "ip": "10.165.115.123",
            "eest": 7
          },
          "ts": 1681909040890
        },
```

Reply:
this is a breaking change for fw apps using modem_info library or nrf_cloud library to update the shadow.